src/components/navMenu/mobileNavMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import {
+  HStack,
+  Text,
+  Link,
+  Icon,
+  Button,
+  useCloseDropDown,
+} from "@nypl/design-system-react-components";
+import React, { useRef, useState } from "react";
+import FocusLock from "@chakra-ui/focus-lock";
+import { Box, chakra } from "@chakra-ui/react";
+import { useScrolled } from "@/hooks/useScrolled";
+
+const listItems = dcNavLinks.map(({ href, text }) => (
+  <Link
+    isUnderlined={false}
+    href={href}
+    key={text}
+    aria-label={text}
+    sx={{
```

Comment:
Not using a `<Text>` element here because it throws the focus ref onto the link, still looking at this

Reply:
I'm a bit confused, did you need to use a `Text` component before? Was it because the link wrapped the `Text` component? That probably throws off the focus and outline.

Sorry, we did not chat about the structure of the nav lists before. They should be structured as
```
<nav aria-label="..."> //every nav on the page should have a unique aria-label
  <ul>
    <li><a ...>...</a></li>
    <li><a ...>...</a></li>
    ...
  </ul>
</nav>
```

Try this instead of the stack components. To get the focus ref align properly, the "li"s can be "empty" (no styles) and the anchor elements should have the padding to expand to the right size.